flow/scripts/load.tcl

```diff
@@ -33,3 +33,59 @@ proc load_design {design_file sdc_file msg} {
     puts $msg
   }
 }
+
+#===========================================================================================
+# Routines to run equivalence tests when they are enabled. 
+
+proc get_verilog_cells_for_design { } {
+    set dir "$::env(PLATFORM_DIR)/work_around_yosys/"
+    set cell_files [glob $dir/*]
+}
+
+proc write_eqy_golden_verilog {} {
+    write_verilog before.v
```

Comment:
This should be written into the results dir not the current dir to avoid collisions.